### PR TITLE
Scroll collt report

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #

### CODE REVIEW

File: interface/reports/collections_report.php [hunk 1]
```diff
@@ -403,6 +415,9 @@ function checkAll(checked) {
 <input type='hidden' name='form_refresh' id='form_refresh' value=''/>
 <input type='hidden' name='form_export' id='form_export' value=''/>
 <input type='hidden' name='form_csvexport' id='form_csvexport' value=''/>
+<input type='hidden' name='form_page_y' id='form_page_y' value=<?php echo $form_page_y; ?>/>
+<input type='hidden' name='form_offset_y' id='form_offset_y' value=<?php echo $form_offset_y; ?>/>
+<input type='hidden' name='form_y' id='form_y' value=<?php echo $form_y; ?>/>
```
Comment (bradymiller) [inline interface/reports/collections_report.php:420]: wrap attr() around each of these echos

Comment (bradymiller) [reply interface/reports/collections_report.php:420]: also should have single quotes around them:
````
<input type='hidden' name='form_y' id='form_y' value='<?php echo attr($form_y); ?>'/>
````

**Comment by stephenwaite** [comment]
testing well and bringing in

**Comment by bradymiller** [comment]
2DV (1)

**Pull request merged**